modules/storage-azure-file-create-storage-class.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage-azure-file.adoc
+
+[id="storage-create-azure-file-storage-class_{context}"]
+= Creating the Azure storage class
+
+StorageClasses are used to differentiate and delineate storage levels and
+usages. By defining a storage class, users can obtain dynamically provisioned
```

Comment:
s/users/you  (unless you're clearly trying to differentiate OCP users from admins.)